src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx

```diff
@@ -0,0 +1,14 @@
+---
+subject: Android agent
+releaseDate: '2022-01-12'
+version: 6.4.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.4.0.zip'
+---
+
+
+## Fixed in this release
+* Resolved D8/R8 warning messages: Expected stack map table for method with non-linear control flow.
+* Fixed a crash casued by NoSuchMethodError happened on Android OS 5 and OS 6.
```

Comment:
Typo: `caused`.

Perhaps rephrase as "_Fixed crash caused by NoSuchMethodError that occurred on Android 5 and 6._"